test/helper.js

```diff
@@ -22,6 +22,7 @@ const resetDatabase = async () => {
   // clear all test data (order matters due to foreign key relationships)
   await models.sequelize.query(`
     DELETE FROM "Users";
+    DELETE FROM "SitePopulations";
```

Comment:
You should also add a `DELETE FROM "Populations"` after this line so that table also gets cleared between test runs...